bridge/shared/interfaces/resourceFileTree.ts

```diff
@@ -0,0 +1,9 @@
+export interface FileTree {
+  stageName: string;
+  tree: TreeEntry[];
+}
+
+export interface TreeEntry {
+  fileName: string;
+  children?: TreeEntry[];
```

Comment:
If we decide to always return `children` the `?` can be removed

Reply:
We leave it as is :)